Dockerfiles/cluster-agent/arm64/Dockerfile

```diff
@@ -43,9 +43,9 @@ COPY --from=builder /output /
 # And we need to do it here because capabilities set in `builder` stage are not kept
 # by the `COPY --from=builder`.
 RUN apt-get update \
-    && apt-get install -y libcap2-bin \
-    && setcap cap_net_bind_service+ep /opt/datadog-agent/bin/datadog-cluster-agent \
-    && apt-get remove -y libcap2-bin \
+    && apt-get install -y attr \
+    && setfattr --name="security.capability" --value="0sAQAAAgAEAAAAAAAAAAAAAAAAAAA=" /opt/datadog-agent/bin/datadog-cluster-agent \
```

Comment:
could you add a comment that explains the meaning of this value?

Reply:
Done